server/config/env/production.js

```diff
@@ -290,7 +290,7 @@ module.exports = {
      *
      */
 
-    trustProxy: process.env.TRUST_PROXY,
+    trustProxy: !!process.env.TRUST_PROXY,
```

Comment:
Boolean-wrapping this environment variable will allow for Sails.js to properly enable or disable the internal flag.